From 66ab3264e3c10e9c7bf4cd2c40eb24eeb9179fac Mon Sep 17 00:00:00 2001
From: Rogerio Guerra Borin <rogerio.borin@toradex.com>
Date: Sat, 24 Aug 2024 15:11:09 -0300
Subject: [PATCH 2/2] Adapt group numbers to match the static assignment in
 torizon

Some Unix groups (input, kvm, sgx) defined in OpenEmbedded layers have
numeric IDs that differ from the ones used by Torizon OS (as defined by
the "torizon-static-group" file) which may lead to users created during
build being assigned to wrong group numbers. Here we fix this situation
by changing them to match the assignment in the torizon-static-group
file; this is also helpful to ensure that group numbers remain the same
as in previous releases of the OS.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Rogerio Guerra Borin <rogerio.borin@toradex.com>
Signed-off-by: Hiago De Franco <hiago.franco@toradex.com>
---
 group.master | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/group.master b/group.master
index 73b6d286c82f..55a77b810a01 100644
--- a/group.master
+++ b/group.master
@@ -12,7 +12,6 @@ uucp:*:10:
 man:*:12:
 proxy:*:13:
 kmem:*:15:
-input:*:19:
 dialout:*:20:
 fax:*:21:
 voice:*:22:
@@ -33,8 +32,6 @@ utmp:*:43:
 video:*:44:
 sasl:*:45:
 plugdev:*:46:
-kvm:*:47:
-sgx:*:48:
 gpio:*:49:
 staff:*:50:
 i2cdev:*:51:
@@ -46,5 +43,8 @@ wheel:*:80:
 nobody:*:99:
 clock:*:81:
 users:*:100:
+input:*:101:
+kvm:*:102:
 render:*:103:
+sgx:*:104:
 nogroup:*:65534:
-- 
2.51.0

